inc/deploygroup_dynamicdata.class.php

```diff
@@ -88,13 +88,60 @@ function getTabNameForItem(CommonGLPI $item, $withtemplate=0) {
 
       if (!$withtemplate
           && $item->fields['type'] == PluginFusioninventoryDeployGroup::DYNAMIC_GROUP) {
-         return array(_n('Criterion', 'Criteria', 2), _n('Associated item','Associated items', 2));
+         $tabs[1] = _n('Criterion', 'Criteria', 2);
+         // Get the count of matching items
+         $count = self::getMatchingItemsCount($item);
+         if ($_SESSION['glpishow_count_on_tabs']) {
+            $tabs[2] = self::createTabEntry(_n('Associated item','Associated items', $count), $count);
+         } else {
+            $tabs[2] = _n('Associated item','Associated items', $count);
+         }
+         return $tabs;
       }
       return '';
    }
 
 
 
+   /**
+    * Get the count of items matching the dynamic search criteria
+    *
+    * @param object $item the item object
+    * @param integer $withtemplate 1 if is a template form
+    * @return string name of the tab
+    */
+   function getMatchingItemsCount(CommonGLPI $item) {
+
+      $params_dyn = array();
```

Comment:
please use [] for array definition instead of array()

Reply:
Ok